drivers/infosilem/campus.cr

```diff
@@ -48,9 +48,7 @@ class Infosilem::Campus < PlaceOS::Driver
 
     @integration.try(&.call(operation: "EndTransfer", body: end_transfer_body(transfer_id: transfer_id.to_s)))
 
-    self["room_#{room_id}_bookings_#{start_date}_#{end_date}"] =
```

Comment:
I think there might be a core bug where when a Module status variable changes at the same time that an exec response occurs, the exec response is replaced by the status change notification message which is something like "status updated module.status_var = {new value}" which is not valid JSON and so the upstream Logic driver reports an error converting the response to JSON. The issue went away when I removed this status update.

Reply:
What can we do about this @stakach ?